src/main/java/io/github/perrymant/moneymaker/Application.java

```diff
@@ -13,6 +13,8 @@
     void start(String[] args) {
         if (args.length > 0 && "help".equals(args[0])) {
             logger.info(reader.read("moneymaker-help.txt"));
+        } else if (args.length > 0 && "report".equals(args[0])) {
```

Comment:
We're checking `args.length` twice, and still print the report if nothing was given. I don't think that's quite intuitive, so let's have a think about what we want to happen for the following cases:

- No arguments given
- `"report"` as the only argument
- `"report"` as the first argument, but there are also other arguments.
- `"help"` as the only argument
- `"help"` as the first argument, but there are also other arguments.
- any other argument that isn't `"report"` or `"help"`

I think this is more or less comprehensive, let me know (in a comment) what you think the behaviour should be.